### PR TITLE
fix py_matcher tutorial

### PR DESCRIPTION
**Pull request by StevenPuttemans**
Fixing issue https://github.com/Itseez/opencv/issues/4237

### CODE REVIEW

File: doc/py_tutorials/py_feature2d/py_matcher/py_matcher.markdown [hunk 1]
```diff
@@ -46,20 +46,20 @@ Here, we will see a simple example on how to match features between two images.
...
 @code{.py}
 import numpy as np
 import cv2
-from matplotlib import pyplot as plt
+import matplotlib.pyplot as plt
 
 img1 = cv2.imread('box.png',0)          # queryImage
 img2 = cv2.imread('box_in_scene.png',0) # trainImage
 
 # Initiate SIFT detector
```
Comment (alalek) [inline doc/py_tutorials/py_feature2d/py_matcher/py_matcher.markdown]: SIFT -> ORB/or remove

**Comment by alalek** [comment]
:+1:

**Comment by StevenPuttemans** [comment]
Auch I did not get the time to merge both fixes. Could you do it post op?

**Comment by alalek** [comment]
Did you mean squashing commits? Don't worry here.

**Comment by StevenPuttemans** [comment]
Yeps :D I do not like screwing up the fixing history :D

**Pull request merged**